src/Voron/StorageEnvironment.cs

```diff
@@ -1037,6 +1037,11 @@ public StorageReport GenerateReport(Transaction tx)
                             case RootObjectType.Table:
                                 countOfTables++;
                                 break;
+                            case RootObjectType.CompactTree:
+                            case RootObjectType.Set:
+                            case RootObjectType.Container:
+                            case RootObjectType.PersistentDictionary:
+                                throw new NotImplementedException($"Not implemented for Corax structures.");
```

Comment:
You created a dedicated exception `NotImplementedInCoraxException`. Didn't you want to use it here?